priority-queue/PriorityQueueConsumerHigh/PriorityQueueConsumerHigh.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" Version="4.1.0" />
```

Comment:
This version is two years old, can we instead target the newer `5.2.0`?

Reply:
Done for all projects

Reply:
Indeed!